Livecheckables/mesa.rb

```diff
@@ -0,0 +1,6 @@
+class Mesa
+  livecheck do
+    url "https://archive.mesa3d.org"
+    regex(/href="mesa-(\d+(?:\.\d+)+)\.t/i)
```

Comment:
```suggestion
    regex(/href=.*?mesa-(\d+(?:\.\d+)+)\.t/i)
```

I missed this on the first pass. We use `href=.*?` these days, as a given HTML page may use `"`, `'`, or even nothing at all (much to my horror).